src/ConversationList/ConversationListItem.vala

```diff
@@ -114,4 +114,45 @@ public class Mail.ConversationListItem : Gtk.ListBoxRow {
 
         return i;
     }
+
+    private string get_relative_datetime (DateTime date_time) {
+        var now = new DateTime.now_local ();
+        var diff = now.difference (date_time);
+
+        if (is_same_day (date_time, now)) {
+            if (diff < TimeSpan.MINUTE) {
+                return _("Now");
+            }
```

Comment:
Use } else if
even if the return doesn't make a difference, it makes if clearer when you read it